src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -30,19 +30,28 @@
 #if TARGET_AUDIOUNIT
 #include "aulayer.h"
 #endif
-/*
-#elif TARGET_VST3
-#include "surgeprocessor.h"
-#elif TARGET_VST2
-#include "vstlayer.h"
-#endif*/
-
-//#include <commctrl.h>
+
+#if MAC || WINDOWS
+#define USE_RUNTIME_LOADED_FONTS 1
+#else
+#define USE_RUNTIME_LOADED_FONTS 0
+#endif
```

Comment:
I mean this :)

Reply:
See comment above. I'm basically following the same way we rolled out zoom. If you feel strongly I can move it to premake, but we've used this convention as we partially roll out OS specific things before, and I think it makes the code nicely readable to have this inline.